pkg/schedule/checker/rule_checker.go

```diff
@@ -404,7 +403,7 @@ func (c *RuleChecker) allowLeader(fit *placement.RegionFit, peer *metapb.Peer) b
 	if s == nil {
 		return false
 	}
-	stateFilter := &filter.StoreStateFilter{ActionScope: "rule-checker", TransferLeader: true}
+	stateFilter := &filter.StoreStateFilter{ActionScope: "rule_checker", TransferLeader: true}
```

Comment:
Will it affect the monitor?

Reply:
<img width="1399" alt="image" src="https://github.com/tikv/pd/assets/23556834/cee523a5-1072-43a0-85e5-08cc292a4b29">

Scope is by group, so it will not affect the monitor. It will generate a new name after the new version goes live, but it won't have any actual impact.

<img width="1205" alt="image" src="https://github.com/tikv/pd/assets/23556834/43148988-3bf1-41f3-bbe5-ad2ecff9d2e8">

Reply:
Will it become xxx_checker-store which mix use dash and underscore?

Reply:
Indeed. We want to use underscores in the code and hyphens for monitoring, right?

Waiting for confirmation~

Reply:
IMO, it's better to use hyphens in monitor.